### ISSUE TITLE
431 Request Header Fields Too Large

### ISSUE THREAD

**Issue by 1129zaj** [nuxt-modules/i18n#2039]
### Environment

"@nuxtjs/i18n": "8.0.0-beta.11"

### Reproduction

"@nuxtjs/i18n": "8.0.0-beta.11"

### Describe the bug

431 Request Header Fields Too Large

**Comment by DenFin** [comment]
I am facing the same issue. I went from 8.0.0-beta.10 to 8.0.0-beta.11 and moved the I18nConfig to `i18n.config.ts` like described [here](https://github.com/nuxt-modules/i18n/commit/f7925d1). When I reload the app in the browser I get the 431 error.

Some additional info:

```
------------------------------
- Operating System: `Darwin`
- Node Version:     `v18.15.0`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `pnpm@8.2.0`
- Builder:          `vite`
- User Config:      `runtimeConfig`, `modules`, `auth`, `app`, `tailwindcss`, `typescript`, `ignore`, `vueI18n`
- Runtime Modules:  `@nuxtjs/tailwindcss@6.2.0`, `@pinia/nuxt@0.4.6`, `@nuxtjs/i18n@8.0.0-beta.11`, `@sidebase/nuxt-auth@0.3.4`
- Build Modules:    `-`
------------------------------
```

**Comment by DenFin** [comment]
For anybody else facing this issue: It has already been mentioned in #2000, #2008 and #2023 and in this [PR](https://github.com/nuxt-modules/i18n/pull/2025)

**Comment by kazupon** [comment]
Thank you for your reporting!

This issue is deplicated with #2008.
This issue will be resolved by #2025 

Thanks!
